### DISCUSSION TITLE
does apache age support undirected graph import which i am making via python networkx library

### DISCUSSION THREAD

**Discussion by prathamgargg** [apache/age#1632] [General]
i am making a graph with python library - **networkx** , which has around 10 million nodes and 20 million edges and it is a normal undirected graph , can i import that into apache age , is it supported ?

**Comment by jrgemignani** [comment]
@prathamgargg Undirected edges aren't supported in Apache AGE. AGE uses digraphs. However, you can roughly have the equivalent by creating 2 edges for each edge, one in each direction.
